### PR TITLE
KOGITO-9645: Create basic integration tests for SWF Chrome extension

### PR DESCRIPTION
**Pull request by tomasdavidorg**
https://issues.redhat.com/browse/KOGITO-9645

Adds basic integration tests for Serverless Workflow Chrome extensions.

### CODE REVIEW

File: packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfFullScreenTest.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+let tools: Tools;
+
+beforeEach(async () => {
+  tools = await Tools.init(TEST_NAME);
+});
+
+test(TEST_NAME, async () => {
+  const processUrl: string =
+    "https://github.com/tomasdavidorg/kie-tools/blob/KOGITO-9644/packages/chrome-extension-serverless-workflow-editor/it-tests/samples/chrome_sample.sw.yaml";
```
Comment (paulovmr) [inline packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfFullScreenTest.ts]: This should refer to the file on kiegroup merged yesterday.

File: packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfTest.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+let tools: Tools;
+
+beforeEach(async () => {
+  tools = await Tools.init(TEST_NAME);
+});
+
+test(TEST_NAME, async () => {
+  const gitHubListPage: GitHubListPage = await tools.openPage(
+    GitHubListPage,
+    "https://github.com/tomasdavidorg/kie-tools/tree/KOGITO-9644/packages/chrome-extension-serverless-workflow-editor/it-tests/samples"
```
Comment (paulovmr) [inline packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfTest.ts]: This should refer to the file on kiegroup merged yesterday.

File: packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfFullScreenTest.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+const TEST_NAME = "SwfFullScreenTest";
+
+let tools: Tools;
+
+beforeEach(async () => {
+  tools = await Tools.init(TEST_NAME);
+});
+
+test(TEST_NAME, async () => {
+  const processUrl: string =
```
Comment (paulovmr) [inline packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfFullScreenTest.ts]:
```suggestion
  const workflowUrl: string =
```

**Review by paulovmr — CHANGES_REQUESTED** [review]

File: packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfTest.ts (hunk 2 above)
Comment (tomasdavidorg) [reply packages/chrome-extension-serverless-workflow-editor/it-tests/tests/SwfTest.ts]: Thanks @paulovmr. I totally forgot to update the links 🤦🏻‍♂️. Sorry 😃 .

**Review by paulovmr — APPROVED** [review]

**Review by domhanak — APPROVED** [review]

**Pull request merged**